Equilobe.DailyReport.BL/BitBucket/BitBucketApiUrls.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Equilobe.DailyReport.BL.BitBucket
+{
+    public class BitBucketApiUrls
+    {
+        public static string PullRequests(string owner, string repository, string page)
```

Comment:
i think it would be better to have the page as an integer, instead of having to call ToString() when calling the method